### PR TITLE
Add GitHub actions for specs

### PR DESCRIPTION
**Pull request by mediafinger**
Hello [plashchynski](https://github.com/plashchynski/crono/commits?author=plashchynski) thanks for creating and maintaining crono.

When you are open to merging these changes, I might propose a few more, like:

* add a _"how-to update to crono v2.0"_ to the README (even if it just explicitly states that nothing has to be changed or done)
* adapt crono to be compatible with Ruby 3.0 (which it currently is not)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ Run the migration:
 
 Now you are ready to move forward to create a job and schedule it.
 
+### Compatibility
+
+* **Crono v1.1.2** and older are compatible with Ruby 2.7.x and older
+* **Crono v2.0.0** and newer are compatible with Ruby 2.7.x and _newer_
```
Comment (mediafinger) [inline README.md:37]: This is kind of untrue, as crono is not yet compatible with Ruby 3.0.

> PS: maybe crono v2.0 is currently compatible with even older Ruby versions, but to ease future development, I would not set this expectation. And given Ruby 2.7 will be EOL soon, it is reasonable to expect people to update their gems when this is easy and they are maintaining their apps.

**Comment by mediafinger** [comment]
Thx, for allowing GitHub Actions, I will make sure they run properly.

**Comment by mediafinger** [comment]
The GitHub Actions to execute the specs work now.

They fail, because the specs fail - also locally for me. I'm going onto holidays from soon, so I won't find the time to fix the specs - or the code to make them pass again.

Any **help in fixing the issues** is welcome.

**Comment by plashchynski** [comment]
Hello @mediafinger 
thank you for your work. I will try to fix the spec

**Pull request merged**